oba/bootstrap.sh

```diff
@@ -5,6 +5,21 @@ NAMESPACE_PREFIX="x"
 NAMESPACE_URI="http://www.springframework.org/schema/beans"
 BEAN_ID="testAPIKey"
 
+# Build bundle inside the container
```

Comment:
I think would be a good idea to avoid overcomplicating our `bootstrap.sh` script, while I haven't learned Perl, I am willing to learn and apply it to the processes.